api/variables/util.go

```diff
@@ -0,0 +1,32 @@
+// Copyright 2024 D2iQ, Inc. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package variables
```

Comment:
WDYT moving these into https://github.com/nutanix-cloud-native/cluster-api-runtime-extensions-nutanix/blob/main/common/pkg/capi/clustertopology/variables/variable.go, seems like that package is already doing something similar related to `clusterv1.ClusterVariable`

Reply:
The rest of the code in this package appears to be dead. I'll check about removing it.

Reply:
Dead code removed in #623